logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -186,11 +187,16 @@ internal class MessageSenderImpl internal constructor(
 
         return target.flatMap { recipients ->
             sessionEstablisher.prepareRecipientsForNewOutgoingMessage(recipients).map { recipients }
-        }.flatMap { recipients ->
+            // map a filtered recipients in case there is an error for x,y,z clients of users
+        }.fold({
+            // if (it is NetworkFailure.FederatedBackendError)
+            // handle federated failure to filter clients and add to QualifiedMessageOption.IgnoreSome
```

Comment:
Is this a TODO or should it be implemented?

Reply:
This was a `TODO`, added the corresponding `TODO(federation)` tag to the comments.